app/jinja_filters.py

```diff
@@ -577,7 +577,7 @@ def map_summary_item_config(
     icon: Optional[str] = None,
 ) -> list[SummaryRow]:
 
-    rows = []
+    rows: list = []
```

Comment:
Changing that results into mypy raising an error `Argument 1 to "extend" of "list" has incompatible type "List[Dict[str, List[Any]]]"; expected "Iterable[SummaryRow]"`. I remember Pete and I tried different ways, like this but it just failing when mypy ran. Should I revert it back to ` rows: list = []`

Reply:
That's because `map_list_collector_config` return a a dict instead of summary row, so this and the return type needs to be `list[Union[dict[str, list], SummaryRow]]`